articles/client-platforms/angularjs/00-intro.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Introduction
+description: test
+---
+
+
+
+<%= include('../../_includes/_signup') %>
+
+Auth0 provides wrappers for using the [Lock widget](https://auth0.com/lock) and the [auth0.js](https://github.com/auth0/auth0.js) library in Anguar apps, and these wrappers will be used throughout these quickstart guides.
+
+## Samples Projects
+
+If you want to follow along with these quickstart guides, download the [seed project](https://github.com/auth0-samples/auth0-angularjs-sample/tree/master/00-Starter-Seed) as a starting point. The seed project is just a basic, blank Angular application with all the Bower dependencies included and the required references added to the `index.html` file. You can also download the source code for each quickstart step from these docs.
+
+## Prerequisites
+
+Before starting, it is important to ensure your application is set up correctly by following these steps:
+
+### 1. Create an Application
+
+<%= include('../../_includes/_new_app') %>_
+
+### 2. Configure Callback URLs
```

Comment:
Let's make these headings h2's instead (##) and without numbers. So this one becomes `## Configure Callback URLs`